packages/ng/src/app/select/select.module.ts

```diff
@@ -1,35 +1,42 @@
 import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
 import { OverlayModule } from '@angular/cdk/overlay';
 import { LuSelectPicker } from './picker';
-import { LuSelectOption } from './option';
+import { LuSelectOption, LuOptionFeederDirective, AbstractSelectOptionFeederComponent } from './option';
 import { LuSelectClearerComponent, LuSelectClearerFirstOrDefaultComponent } from './clearer';
 import { LuSelectDirective } from './directive';
+import { LuSelectSearcherComponent} from './searcher';
 import { LuSelect } from './select.component';
 import { LuEmptyModule } from '../empty/empty.module';
 import { LuPopoverModule } from '../popover/popover.module';
 
 @NgModule({
 	imports: [
 		CommonModule,
+		FormsModule,
 		OverlayModule,
 		LuPopoverModule,
 		LuEmptyModule,
 	],
 	declarations: [
 		LuSelectPicker,
 		LuSelectOption,
+		LuOptionFeederDirective,
```

Comment:
why do you declare components in this module when this module should just be a conglomerate of `SelectOptionModule`, `SelectOptionFeederModule` `SelectClearerModule` etc...

Reply:
i say that cuz i try to fix `ng build -prod` and see this error
```
RROR in : Type LuSelectOption in/src/app/select/option/select-option.component.ts is part of the declarations of 2 modules: LuSelectModule in /src/app/select/select.module.ts and LuSelectOptionModule in /src/app/select/option/select-option.module.ts!
```